package/src/BuyCryptoView/TopScreenB2/CurrencySwitcher/CurrencySwitcher.tsx

```diff
@@ -44,19 +46,36 @@ const CurrencySwitcher: React.FC = () => {
   const { t } = useTranslation();
   const { collected, data } = useContext(APIContext);
   const { nextScreen, backScreen } = useContext(NavContext);
+  const sendDataToGTM = useGTMDispatch();
 
   const [pair, setPair] = useState<
     ({ item?: ItemType; hasOptions: boolean } | undefined)[]
   >([]);
 
   const handleItemClick = useCallback(
     (name: string, index: number, item: ItemType) => {
-      if (name === "crypto") data.handleCryptoChange(item);
-      else if (name === "currency") data.handleCurrencyChange(item);
-
+      let gtmData;
+      if (name === "crypto") {
+        gtmData = {
+          event: GtmEvent.ELEMENT_CLICK,
+          action: GtmEventAction.OUT_CURRENCY_SELECTION,
+          category: GtmEventCategory.DROPDOWN_VALUE,
+          label: item.name,
+        };
+        data.handleCryptoChange(item);
+      } else if (name === "currency") {
```

Comment:
rename this to fiat

Reply:
It's existing code. I only added gtm related changes. gtm object & dispatcher